src/mongo/db/ldap_options.idl

```diff
@@ -118,4 +126,13 @@ configs:
         arg_vartype: String
         validator:
             callback: validateLDAPUserToDNMapping
-
+    'security.ldap.debug':
+        description: 'Print debug information for LDAP connections'
+        short_name: ldapDebug
+        arg_vartype: Bool
+        default: false
+    'security.ldap.follow_referrals':
+        description: 'Automatically follow LDAP referrals with the same bind credentials'
+        short_name: ldapReferrals
+        arg_vartype: Bool
+        default: false
```

Comment:
In the pull request comment you wrote ` security.ldap.follow_referrals,
which defaults to true` but here it is `false`. I guess code is correct here but please check it.

Reply:
Yes, commit message edited.

Reply:
Oh, i see, commit message has changed but pull request's main comment still the same